auction-app-frontend/src/pages/Home/style.scss

```diff
@@ -0,0 +1,121 @@
+@import "src/scss/index.scss";
+
+.home-container {
+  margin: 80px auto 0 auto;
+
+  .home-upper {
+    @include flex-row;
+    justify-content: space-between;
+    align-items: center;
+    align-content: center;
+    height: 420px;
+
+    .categories {
+      flex: 0 0 262px;
+
+      .categories-heading {
+        color: $primary;
+        padding-bottom: 26px;
+        text-transform: uppercase;
+      }
+
+      ul {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+      }
+
+      li {
+        margin-bottom: 10px;
+        cursor: pointer;
+
+        &:after {
+          content: "";
+          display: block;
+          height: 1px;
+          width: 100%;
+          background-color: $neutral09;
+          margin-top: 10px;
+        }
+      }
+    }
+
+    .highlighted-product {
+      @include flex-row;
+      align-items: flex-start;
+      width: 739px;
+      height: 420px;
+      flex: 1;
+
+      .product-image {
+        height: 100%;
+
+        img {
+          max-height: 100%;
+          width: 357px;
+          object-fit: cover;
+        }
+      }
+
+      .product-container {
+        @include flex-row;
+        justify-content: center;
+        gap: 36px;
+        height: 420px;
+        flex: 1;
+
+        .product-info {
+          @include flex-column;
+          width: 346px;
+          gap: 16px;
+
+          p {
```

Comment:
Instead reseting margin for `p` tag, can this be for example span with some styling you need.